src/templates/zh-TW/2020/methodology.html

```diff
@@ -0,0 +1,476 @@
+<span class="comment"># 01_01b: Distribution of JS bytes by client</span>
+<span class="keyword">SELECT</span>
+  percentile,
+  _TABLE_SUFFIX <span class="keyword">AS</span> client,
+  <span class="function call">APPROX_QUANTILES</span>(<span class="function call">ROUND</span>(bytesJs / <span class="constant numeric">1024</span>, <span class="constant numeric">2</span>), <span class="constant numeric">1000</span>)[<span class="function call">OFFSET</span>(percentile <span class="keyword operator">*</span> <span class="constant numeric">10</span>)] <span class="keyword">AS</span> js_kbytes
+<span class="keyword">FROM</span>
+  <span class="string">`httparchive.summary_pages.2019_07_01_*`</span>,
+  <span class="function call">UNNEST</span>([<span class="constant numeric">10</span>, <span class="constant numeric">25</span>, <span class="constant numeric">50</span>, <span class="constant numeric">75</span>, <span class="constant numeric">90</span>]) <span class="keyword">AS</span> percentile
+<span class="keyword">GROUP</span> <span class="keyword">BY</span>
+  percentile,
+  client
+<span class="keyword">ORDER</span> <span class="keyword">BY</span>
+  percentile,
+  client</code></pre>
+        </div>
+
+        <p>
+          每個指標的結果可以在各個章節對應的電子表格中公開查詢，例如<a href="https://docs.google.com/spreadsheets/d/1kBTglETN_V9UjKqK_EFmFjRexJnQOmLLr-I2Tkotvic/edit?usp=sharing">JavaScript結果</a>。滑動到各個章節的底部，你可以看到查詢、結果、和讀者評論的連結按鈕。
+        </p>
+    </section>
+
+    <section>
+        <h3 id="websites"><a href="#websites" class="anchor-link">網站</a></h3>
+
+        <p>
+          資料集共有7,546,709個網站，其中6,347,919個是手機版網站與5,593,642個是桌機版網站。大多數的網站都同時是桌機版和手機版的子集合。
+        </p>
+
+        <p>
+          HTTP Archive從<a href="#chrome-ux-report">Chrome UX Report</a>獲取網站的URLs。Chrome UX Report是Google的公共資料集，集合Chrome活躍使用者訪問數以百萬網站的使用者經驗，這些網站列表反映著最新的真實網站使用狀況。Chrome UX Report資料集透過一個因子維數，取得所有桌機版與手機版使用者訪問的網站。
+        </p>
+
+        <p>
+          Web Almanac網路年鑑爬取2020年8月HTTP Archive的資料集，網站列表則是透過最近期202006的Chrome UX Report，在2020年7月14日發布，擷取6月分Chrome使用者瀏覽的網站。
+        </p>
+
+        <p>
+          2020年被分析的網站有20-30%的增長，相較於<a href="../2019/methodology#websites">2019年Web Almanac網路年鑑</a>。你可閱讀這篇文章，<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='paulcalvano') }}">Paul Calvano</a>的分析：<a href="https://paulcalvano.com/2020-09-29-growth-of-the-web-in-2020/">2020年網路使用的增長</a>。
+        </p>
+
+        <p>
+          礙於資源的限制，HTTP Archive只能測試Chrome UX report每個網站的單一網頁，協調平衡這點後我們只包括網站的主頁。這結果將導致些許偏差，因為一個主頁不能絕對地代表一整個網站。
+        </p>
+
+        <p>
+          HTTP Archive也被考慮成一個測試工具，它是透過數據中心測試網站，並不是從真實世界蒐集使用者經驗。所有的測試都基於一個處於登出狀態的空白快取主頁，這可能不能真實反映使用者是如何訪問它們。
+        </p>
+    </section>
+
+    <section>
+      <h3 id="metrics"><a href="#metrics" class="anchor-link">指標</a></h3>
+
+      <p>
+        HTTP Archive蒐集數千個指標用來衡量網路是如何被建構。它包括基本的指標，像是網頁使用的位元數，網頁加載是否透過HTTPS，個別的請求，標頭反應。這些主要的指標都是由<a href="#webpagetest">WebPageTest</a>提供，它為每個網站提供測試路徑。
```

Comment:
列舉各種項目時用頓號（、）也許比較正確 👀?

```suggestion
        HTTP Archive 蒐集數千個指標用來衡量網路是如何被建構。它包括基本的指標像是網頁使用的位元數、網頁加載是否透過 HTTPS、個別的請求以及標頭反應。這些主要的指標都是由<a href="#webpagetest">WebPageTest</a>提供，它為每個網站提供測試路徑。
```

Reply:
同意喔~